GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1803,6 +1803,11 @@ private void PopulateRecentRepositoriesMenu(ToolStripDropDownItem container)
 
             void AddRecentRepositories(Repository repo, string caption)
             {
+                if (AppSettings.RecentReposShowBranchName)
+                {
+                    caption = $"{caption} [{GitModule.GetSelectedBranchFast(repo.Path)}]";
+                }
+
                 if (!string.IsNullOrEmpty(repo.Category))
                 {
                     caption = $"{caption} ({repo.Category})";
```

Comment:
As a side note

😕 I don't believe this piece of code is working at all.
The collection of recent repositories doesn't have "category" set, only the collection of favourites does, but it isn't bound here...

Reply:
You are right, category is always null here